src/main/java/com/linkedin/kmf/services/ClusterTopicManipulationService.java

```diff
@@ -45,22 +53,33 @@
   private final ScheduledExecutorService _executor;
   private final AdminClient _adminClient;
   private boolean _isOngoingTopicCreationDone;
+  private boolean _isOngoingTopicDeletionDone;
```

Comment:
Just curious: is topic update in the picture?

Reply:
Yup, it is whether or not the operation for Topic Deletion (Update) is complete since it's been first issued.